phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexCoprocIT.java

```diff
@@ -58,12 +58,12 @@ public class IndexCoprocIT extends ParallelStatsDisabledIT {
         "|org.apache.phoenix.index.GlobalIndexChecker|805306365|";
     public static final String INDEX_REGION_OBSERVER_CONFIG =
         "|org.apache.phoenix.hbase.index.IndexRegionObserver|805306366|" +
-            "index.builder=org.apache.phoenix.index.PhoenixIndexBuilder," +
```

Comment:
index.builder and org.apache.hadoop.hbase.index.codec.class are present in the HBase table descriptor.
For data backwards compatibility reasons I'd prefer not change these two (even though they have the same problems as others).

i.e. Changing these would break older versions of Phoenix when trying to use them if we changed these.

Reply:
And these are not read from hbase-site.xml, but from the Hbase table metadata directly.

Reply:
Thank you for pointing this out, will restore the original strings. 
The string "index.builder" was declared twice though, I think one of them was unnecessary.

Reply:
Can you revert the whitespace changes ?
If we don't change these, it's better to leave the file alone.